packages/gatsby-theme/src/components/Contact/Contact.js

```diff
@@ -1,17 +1,34 @@
 import React from "react"
 import PropTypes from "prop-types"
 import { Box } from "theme-ui"
+import { RichText } from "ui"
 
-const Contact = ({ children, ...props }) => {
+const Contact = ({ children, title, align, sx, ...props }) => {
   return (
-    <Box variant='wrapper' __themeKey='container' {...props}>
+    <Box
+      variant='wrapper'
+      __themeKey='container'
+      sx={{ textAlign: align, ...sx }}
+      {...props}
+    >
+      <RichText sx={{ textAlign: align, pb: "small", ...sx }} {...props}>
+        {title}
+      </RichText>
       {children}
     </Box>
   )
 }
 
 Contact.propTypes = {
   children: PropTypes.node.isRequired,
+  title: PropTypes.string.isRequired,
```

Comment:
Maybe we could change the `required` flag for the title to avoid issues with the block